### PR TITLE
Update docs comment for start/stoptrackpage

### PR DESCRIPTION
**Pull request by markwolff**
Please fill in this template.

- [X] Use a meaningful title for the pull request. Include the name of the package modified.
- [X] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Microsoft/ApplicationInsights-JS/issues/447
- [x] Increase the version number in the header if appropriate.
- [x] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/applicationinsights-js/index.d.ts [hunk 1]
```diff
@@ -670,13 +670,15 @@ declare module Microsoft.ApplicationInsights {
         context: ITelemetryContext;
         queue: Array<() => void>;
         /**
-         * Starts timing how long the user views a page or other item. Call this when the page opens.
-         * This method doesn't send any telemetry. Call {@link stopTrackTelemetry} to log the page when it closes.
+         * Starts the timer for tracking a page load time. Use this instead of `trackPageView` if you want to control when the page view timer starts and stops,
+         * but don't want to calculate the duration yourself. This method doesn't send any telemetry. Call `stopTrackPage` to log the end of the page view
```
[the remark this answers is not in the document]
Comment (markwolff) [reply types/applicationinsights-js/index.d.ts]: Doesn't render on tooltips for some IDE's (e.g. https://github.com/Microsoft/vscode/issues/57495). It caused visual clutter in my opinion. Open to suggestions

[the remark this answers is not in the document]
Comment (amcasey) [reply types/applicationinsights-js/index.d.ts]: I'll defer to the owners.  @jpiyali @barustum?

[the remark this answers is not in the document]
Comment (amcasey) [reply types/applicationinsights-js/index.d.ts]: Great.  I'll merge as soon as you drop the broken commit, @markwolff.

[the remark this answers is not in the document]
Comment (markwolff) [reply types/applicationinsights-js/index.d.ts]: @amcasey Reverted to backticks

**Pull request merged**